Code/Editor/Plugins/ComponentEntityEditorPlugin/UI/QComponentEntityEditorMainWindow.cpp

```diff
@@ -21,6 +21,7 @@ QComponentEntityEditorInspectorWindow::QComponentEntityEditorInspectorWindow(QWi
     : QMainWindow(parent)
     , m_propertyEditor(nullptr)
 {
+    setObjectName("ComponentEntityInspector");
```

Comment:
UX is actually planning to change the name of this widget from "Entity Inspector" to just "Inspector" (since it is switching to also showing prefab properties, level etc).
We don't need to do the rename work here, but since this is being introduced I'd use a more generic name so we don't have to change it later (either just "Inspector", or "InspectorWidget"?)

Reply:
Good point! "Inspector" and "InspectorWidget" are both taken by Atom already, so I'll go with "InspectorMainWindow" instead, since this class is actually subclassing `QMainWindow`